app/controllers/idv/unavailable_controller.rb

```diff
@@ -0,0 +1,40 @@
+module Idv
+  class UnavailableController < ApplicationController
+    def show
+      if FeatureManagement.idv_available?
+        if from_create_account?
+          return redirect_to sign_up_email_url
+        else
+          return redirect_to account_url
+        end
+      end
+
+      show_without_redirect
+    end
+
+    def show_without_redirect
+      analytics.vendor_outage(
+        vendor_status: {
+          acuant: IdentityConfig.store.vendor_status_acuant,
+          lexisnexis_instant_verify: IdentityConfig.store.vendor_status_lexisnexis_instant_verify,
+          lexisnexis_trueid: IdentityConfig.store.vendor_status_lexisnexis_trueid,
+          sms: IdentityConfig.store.vendor_status_sms,
+          voice: IdentityConfig.store.vendor_status_voice,
+        },
+        redirect_from: from,
+      )
+      render 'idv/unavailable', status: :service_unavailable
```

Comment:
Two thoughts which may allow us to remove this line altogether:

- Could we name the view conventionally so that Rails would auto-discover it? I think it'd also make it more discoverable for future maintainers.
   - i.e. `app/views/idv/unavailable/show.html.erb`
- Maybe debatable, but I personally don't feel that this warrants a 5XX, since we _are_ handling the request successfully. I feel like a 200 would be perfectly reasonable.

Reply:
Agreed on using standard action names and discoverable view locations! I'd like to keep the 5XX since I think it captures the spirit of the error and expresses it semantically.

Reply:
Would there be knock-on effects of rendering a 5XX? Like setting off some of our alerting and paging folks?

Reply:
Answer: yes, there would be! 200 it is.